service/src/main/java/uk/nhs/adaptors/oneoneone/cda/report/mapper/HealthcareServiceMapper.java

```diff
@@ -23,6 +26,8 @@
 @Component
 @RequiredArgsConstructor
 public class HealthcareServiceMapper {
+
+    private static final String PRCP = "PRCP";
```

Comment:
More descriptive name would be nice, for example `PRCP _TYPE_CODE` or something similar.